spec/lib/reverse_markdown/cleaner_spec.rb

```diff
@@ -14,6 +14,13 @@
       expect(result).to eq "foo\nbar\n\nbaz"
     end
   end
+  
+  describe '#remove_leading_whitespaces' do
+    it 'removes spaces after line breaks' do
+      result = cleaner.remove_leading_whitespaces("foo\r bar")
```

Comment:
@krisdigital Hi there, just browsing through all the open PRs and try to close as many as possible :)

Under which circumstances do you find a solo "\r" within an HTML doc? I'd rather remove it entirely (since it has even less use within a markdown doc) than to remove the whitespace.

So I think the test should be like this: `cleaner.tidy("foo\r bar") => "foo bar"`

What do you think?

Reply:
@xijo it has been a while, but like in the spec a faced the problem that the markup I got from a popular CMS lead to whitespace at the beginning of lines like in the spec example:

```
expect(ReverseMarkdown.convert("Praia Joaquina \r\n\r\nThe spot")).to eq "Praia Joaquina \r\rThe spot\n\n"
```
As long as that case is fixed every solution works for me ;)